octoprint_mrbeam/analytics/analytics_handler.py

```diff
@@ -896,7 +913,12 @@ def _cleanup_job(self):
     def _init_new_job(self):
         self._cleanup_job()
         self._current_job_id = "j_{}_{}".format(self._snr, time.time())
-        self._add_job_event(ak.Job.Event.LASERJOB_STARTED)
+        payload = {
+            ak.Device.LaserHead.HEAD_MODEL_ID: self._plugin.laserhead_handler.get_current_used_lh_data()[
+                "model"
+            ],
+        }
+        self._add_job_event(ak.Job.Event.LASERJOB_STARTED, payload=payload)
```

Comment:
Personal taste, but this would have been a good place to suppress the black formatter
```py
# fmt: off
some code that was auto formatted into uglyness
# fmt: on
```

Reply:
Yes, I will add this